README.md

```diff
@@ -10,55 +10,54 @@ what can be done with modern software engineering tools.
 * A working pipeline will also demonstrate a continuous delivery process by automatically promoting your application to a live, production environment within minutes of committing code to a source control repository.
 
 ###How You'll Do It
-1. Break up into small teams of 2-4 individuals
-2. Fork and clone Application code from GitHub
-3. Get the application working and commit it back to GitHub
-4. See your app deploy to an app server
+1. Break up into small teams of 2-4 individuals.
+2. Fork and clone application code from GitHub.
+3. Get the application working and commit it back to GitHub.
+4. See your app deploy to an app server.
 
 ###Step 1: Organize
 1. Break up into small teams of 2-4 people.
-2. Look at the failing build at http://build.liatrio.com/....(Which build server are we using for this?) //NEED TO UPDATE
+2. Look at the failing build at [jenkins.chico.liatr.io](https://jenkins.chico.liatr.io)
 3. Review the documentation and decide how to proceed.
+4. Install project dependencies that are required to build/test/run your application.
+   * Select your OS from the list of README's below then return to this page after installion is complete.
+      * [Mac OS X using Homebrew](Other-READMES/mac-homebrew-setup.md) _*Recommended_.
+      * [Mac OS X manual install](Other-READMES/mac-manual-setup.md)
+      * [Ubuntu](Other-READMES/ubuntu-env-setup.md)
 
 ###Step 2: Forking, Cloning and Building Your Application
 1. Get a copy of the code for the exercise.
    1. Fork the application repository at github.com/liatrio/game-of-life to one or more of your team members' Github Accounts.
    2. If you don't have a Github account you should create one now.
 2. See your repository build on our build server.
    1. Browse to [jenkins.chico.liatr.io](jenkins.chico.liatr.io) and sign in using the information given to you.
-   2. Create a new Jenkins job with your repository
+   2. Create a new Jenkins job with your repository.
       1. Click _New Item_ on the left hand side.
-      2. Enter the name of your job
-      3. Click _Maven Project_
+      2. Enter the name of your job.
+      3. Click _Maven Project_.
       4. Scroll down to _Source Code Management_, select _Git_, and enter the link to your repository.
       5. Scroll down to _Build Triggers_ and click _Poll SCM_ and enter "* * * * *"
       6. Scroll down to _Build_  and add "clean install" to _Goals and Options_
-      7. Click _Save_
-   4. Verify that your job is added to Jenkins start a build.
-      1. It should fail like the original application.
+      7. Add a _Post Step_ and insert the follow: ```scp -i /var/lib/jenkins/.ssh/id_rsa -o StrictHostKeyChecking=no \
```

Comment:
We want to remove this step.  We won't have it deploy until automation is added.